### PR TITLE
terraform-docs 0.7.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/terraform-docs.rb [hunk 1]
```diff
@@ -15,15 +15,9 @@ class TerraformDocs < Formula
...
-    ENV["GOPATH"] = buildpath
-    dir = buildpath/"src/github.com/segmentio/terraform-docs"
-    dir.install buildpath.children
-
-    cd dir do
-      system "make", "build-darwin-amd64"
-      bin.install "bin/darwin-amd64/terraform-docs"
-      prefix.install_metafiles
-    end
+    system "make", "build-darwin-amd64"
```
Comment (khos2ow) [inline Formula/terraform-docs.rb]: This has changed, simply the following is enough.
```suggestion
    system "make", "build"
```

**Review by khos2ow — COMMENTED** [review]
This is my first time contributing and I actually don't know how this PR got created, I was trying `brew bump-formula-pr` but it kept failing!

File: Formula/terraform-docs.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/terraform-docs.rb]: Thanks!

**Comment by chenrui333** [comment]
@BrewTestBot test this please

**Comment by chenrui333** [comment]
@BrewTestBot test this please

**Pull request closed without merging**